fluent-bundle/src/bundle.rs

```diff
@@ -57,10 +57,7 @@ pub struct FluentBundle<'bundle> {
 
 impl<'bundle> FluentBundle<'bundle> {
     pub fn new<'a, S: ToString>(locales: &'a [S]) -> FluentBundle<'bundle> {
-        let locales = locales
-            .into_iter()
-            .map(|s| s.to_string())
-            .collect::<Vec<_>>();
+        let locales = locales.iter().map(|s| s.to_string()).collect::<Vec<_>>();
```

Comment:
Should FluentBundle.{locales,entries,plural_rules} be made private as part of this change?

Reply:
Seems like `entries` and `plural_rules` need to stay public to be usable from outside of the struct. We could limit `locales`, but I'm wondering if we should invest in some API design.